addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -0,0 +1,28 @@
+from odoo.tests.common import TransactionCase
+from odoo.tests import tagged
+from odoo.tools.parse_version import parse_version
+import odoo.addons.phone_validation.lib.phonenumbers_patch
+
+@tagged('post_install', '-at_install')
+class TestPhonenumbersPatch(TransactionCase):
+    @classmethod

```

Comment:
```suggestion
class TestPhonenumbersPatch(BaseCase):
```
* the tagged doesn't seem needed to me: this is a python modification so we should be able to test it at any time
* I don't think there should be any sql run so `TransactionCase` doesn't seem needed.
* I'm not sure why we are doing `@classmethod`